contrib/udfs/src/test/java/org/apache/drill/exec/udfs/TestNearestDateFunctions.java

```diff
@@ -149,7 +149,7 @@ public void testReadException() throws Exception {
       run(query);
       fail();
     } catch (DrillRuntimeException e) {
-      assertTrue(e.getMessage().contains("[BAD_DATE] is not a valid time statement. Expecting: " + Arrays.asList(NearestDateUtils.TimeInterval.values())));
+      assertTrue(e.getMessage().contains("[BAD_DATE] is not a valid time statement. Expecting: " + Arrays.asList(DateConversionUtils.TimeInterval.values())));
```

Comment:
Is TestNearestDateFunctions still a good name for this class given the renamings made in this PR?

Reply:
Renamed...